docs/quick-tutorials/k8s-kafka-mtls.mdx

```diff
@@ -10,21 +10,14 @@ automating the management of Kafka ACLs, and the generation and deployment of ce
 
 In this tutorial, we will:
 
-- Deploy a Kafka broker, and three clients that call it.
-- Discover the topic-level access those clients require, using the Otterize network mapper's Kafka watcher.
-- Declare that one client pod intends to access a topic on Kafka.
-- See that an ACL was autogenerated to allow just that, while blocking calls from the other client.
-- See that mTLS credentials were autogenerated.
+- Deploy a Kafka broker with 2 topics, and two clients that call these topics.
+- Protect just 1 of those topics with Otterize. See that access to that topic is blocked.
+- Declare that one client pod intends to access that protected topic.
+- See that an ACL was autogenerated to allow just that, while blocking calls to that topic from the other client.
+- Also verify that mTLS credentials were autogenerated for all clients.
```

Comment:
```suggestion
```

Reply:
I think we can remove this bit from the tutorial entirely, I believe it's pretty self-explanatory. We do explain that it goes in a Kubernetes Secret, that's enough IMO.